notebooks/4.visualization/nbconverted/4.visualization.r

```diff
@@ -22,19 +22,19 @@ load_image <- function(path) {
 
 # getting all paths
 # confusing matrix paths
-confusion_matrix_path <- file.path("../../results/2.modeling/confusion_matrix.csv.gz")
+confusion_matrix_path <- file.path("../../results/2.modeling/fs_confusion_matrix.csv.gz")
```

Comment:
please make sure all of the original files (without the `fs` prefix) are no longer in the repo

Reply:
Just looked at all the results folder and all files have the `fs_` and the `aligned` prefix